### DISCUSSION TITLE
Need to click twice to open from task bar

### DISCUSSION THREAD

**Discussion by THRobinson75** [RandomEngy/VidCoder#1322] [Q&A]
All icons on my task bar, ie. chrome, outlook, etc... when running all I need to do is click once and it opens up.

New PC, now running Win11... Vicoder is encoding, and when I click the icon, it DINGS, then when I click again, it opens.

Only Vidcoder, the other 8 icons on my taskbar don't do this.

Any idea why?

**Comment by THRobinson75** [comment]
Anyone? am I the only one with this issue?

**Comment by Lobo204** [comment]
Nope, I have had this for a while. I don't know if it was either Vidcoder or a Windows 11 Update that caused it.

**Comment by THRobinson75** [comment]
Ya, I just built a new PC, and when I installed the software I went with Win11 and downloaded latest VidCoder so, basically upgraded both at once so no idea which is doing it.

**Comment by RandomEngy** [comment]
This is fixed in v11.10.

Looks like an operation to clear the notification history on window activation was preventing the window restore. I delayed the notification clear operation a bit and now it seems to be working for me.
